msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/nestedAppAuth/NestedAppHelper.java

```diff
@@ -181,6 +187,7 @@ protected void performATSilentForNestedApp(AccountRecord accountRecord, boolean
         if (shouldAddDeviceIdClaim) {
             claimsJsonString = DEVICE_ID_CLAIM;
         }
+        final String hubAppClientId = mLabAccount.getUserType() == UserType.CLOUD ? HUB_APP_CLIENT_ID_AAD : HUB_APP_CLIENT_ID_MSA;
```

Comment:
I guess perform similar check as on line 112? So check for MSA and then return MSA id otherwise AAD

This way anything other than MSA can fall under AAD bucket (such as a federated or guest UserType)